tests/arc-validate.Tests/ARCSpecification.fs

```diff
@@ -0,0 +1,27 @@
+﻿module ARCSpecification
+
+open Expecto
+open System.IO
+open ARCTokenization
+
+
+[<Tests>]
+let tests =
+    testList "ARCValidationPackage tests" [
+        test "V2_Draft" {
+
+            let testPath = System.IO.Path.GetFullPath("fixtures/arcs/testingArc/")
+
+            let cases = ARCValidate.ARCSpecification.V2_Draft.testCases testPath
+
+            
+            let runSummery = ARCExpect.Execute.Validation cases
+
+            //false None (ParseResults<ARCValidate.CLIArguments.ValidateArgs>.Empty)
+            Expect.isTrue
```

Comment:
As you can see in the case of the other tests, in this test project we do not test library functions but the execution of the tool. In this case, a test for `arc-validate validate` is needed. This test however is completely valid once it has been moved into `ARCExpect.Tests`